bin/getTitle.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:http/http.dart' as http;
```

Comment:
As I explained in this [slide](https://docs.google.com/presentation/d/1tybmc0YPWrdmRy72DM4TAvJq9HPVocZgZB-k9tEdNRo/edit#slide=id.g855d8ed3b8_1_28), each exercise should be done in a separate folder.

The reason is that each exercise is a different program. For example, the exercise that displays the whole HTML code does not need the "html" package. So in its pubspec.yaml, you should not contain "html". Also they could have different names and different README.md to explain what the project does.

Also, the convention in Dart is for the main program to be in bin/main.dart. This way, we can run any Dart program by running `dart bin/main.dart`. In your case, you called your file `getTitle.dart`, which would be confusing to someone who doesn't know your project.

Can you create two folders, and separate the two exercises?

Reply:
Thank you for your kind explanation. I will create two folders and separate these two exercises.